root/cdtoc/attach_list.tt

```diff
@@ -18,18 +18,25 @@
   </tr>
   [% FOR medium=release.mediums %]
     [% NEXT UNLESS
-        medium.cdtoc_track_count == cdtoc.track_count %]
+        medium.cdtoc_track_count == cdtoc.track_count;
+       this_medium_has_cdtoc = medium_has_cdtoc.defined AND (medium.id == medium_has_cdtoc) %]
```

Comment:
Indent seems off?

Reply:
The line I added looks okay since it's lined up with the `NEXT`. The one above should probably be indented +1 space, but not sure I care enough to fix old TT. :P